services/editor/src/components/App.js

```diff
@@ -32,9 +33,11 @@ const App = ({ children }) => (
 
 const preload = async () => await Promise.all([TypesService.refreshTypes(), refreshSchema()]);
 
+const loadingRenderer = () => null;
+const loadingErrorRenderer = error => <ErrorPage error={error} />;
```

Comment:
the name is not clear. it implies that this is loading the error renderer. you can just call it `errorRenderer`.